packages/gcf-utils/package.json

```diff
@@ -43,7 +43,6 @@
     "@types/sinon": "^9.0.0",
     "@types/tmp": "^0.2.0",
     "@types/yargs": "^15.0.0",
-    "@octokit/rest": "^16.43.1",
```

Comment:
because of the backflips being done by `release-please` to use octokit@18, even though probot is pinning us to 16, adding this dep causes issues with tests.